operations/flipt-in-production.mdx

```diff
@@ -0,0 +1,86 @@
+---
+title: Flipt In Production
+description: This document describes recommended configuration options for operating Flipt in Production.
+---
+
+There will eventually come a time where you are ready to use Flipt in a production setting against real users, after having used it in lower testing environments. Naturally, production environments tend to have higher traffic rates than lower environments which in turn, _could_ impact Flipt instance(s).
+
+Here are some configuration options and tips to consider when operating Flipt in production:
+
+1. **Database connection limits**
+
+By default the Golang `database/sql` client, will have `MaxOpenConn` equal to 0 (unlimited), and `MaxIdleConn` equal to 2.
+
+With the databases that listen over a network (MySQL, Postgres, CockroachDB), there are default server limits for the number of open connections it supports.
+
+In high burst traffic scenarios, this can lead to the `too many open connections` error server side.
+
+You should tweak that number to be above 0, and to whatever fits your use case. This can be done via environment variable:
```

Comment:
```suggestion
You should tweak that number to be above 0, and to whatever fits your use case. This can be done via setting an environment variable:
```